content/post/yet-another-reason-to-privatize-the-police.md

```diff
@@ -34,21 +34,21 @@ This brings us to the second major problem with the state-funded police: the pro
+Business insurers clearly have plenty of skin in the game when it comes to preventing mass property damage and theft to their policyholders. No insurance provider benefits from having to pay out claims; and after having witnessed the utter failure of state-funded police to perform their most basic duties, insurance providers may have all the incentive they need to start offering their own private protection and law enforcement.
 
 Aside from the state's monopoly on police provision, nothing stands in the way of a market solution pioneered by insurance companies that allows individuals and businesses to purchase security insurance policies for property protection and defense against criminal behavior. Would such a system be so different from how we subscribe to, and seamlessly switch between, auto-insurance providers and different levels of coverage tailored to our specific needs? 
 
-In terms of commercial security policies, businesses in remote areas may find it beneficial to purchase an individual security policy for the protection of only that business's premises. In more crowded locations it may be better for businesses and/or apartment complexes located within a particular city block, street, or district to pool their funding and purchase a group security policy to prevent criminal activity and protect personal property across a wider area. When it comes to residential protection, members of homeowners associations (HOAs) can, and already do, pay monthly dues for [private HOA security teams](http://bravothree.com/homeowners-association-security/) to ensure that their residential areas are properly patrolled for criminal activity and that residents adhere to the laws and regulations of the HOA to which they voluntarily subscribe. Why would insurance companies be any less capable of expanding similar, already existing private services into commercial areas?
+In terms of commercial security policies, businesses in remote areas may find it beneficial to purchase an individual security policy for the protection of only that business's premises. In more crowded locations it might be more cost-effective for businesses and apartment complexes located within a particular city block, street, or district to pool their funding and purchase a group security policy to prevent criminal activity and protect personal property across a wider area. When it comes to residential protection, members of homeowners associations (HOAs) can, and already do, pay monthly dues for [private HOA security teams](http://bravothree.com/homeowners-association-security/) to ensure that their residential areas are properly patrolled for criminal activity and that residents adhere to the laws and regulations of the HOA to which they voluntarily subscribe. Why would insurance companies be any less capable of expanding similar services into commercial areas?
```

Comment:
```suggestion
Businesses in remote areas may find it beneficial to purchase an individual security policy for the protection of only that business's premises. In more crowded locations it might be more cost-effective for businesses and apartment complexes located within a particular city block, street, or district to pool their funding and purchase a group security policy to prevent criminal activity and protect personal property across a wider area. When it comes to residential protection, members of homeowners associations (HOAs) can, and already do, pay monthly dues for [private HOA security teams](http://bravothree.com/homeowners-association-security/) to ensure that their residential areas are properly patrolled for criminal activity and that residents adhere to the laws and regulations of the HOA to which they voluntarily subscribe. Why would insurance companies be any less capable of expanding similar services into commercial areas?
```